ga/latest/kernel/Dockerfile.ubi.ibmjava8

```diff
@@ -19,10 +19,10 @@ USER root
 ARG VERBOSE=false
 
 # Install WebSphere Liberty
-ARG LIBERTY_VERSION=24.0.0.12
+ARG LIBERTY_VERSION=25.0.0.1
 ARG LIBERTY_URL
 ARG DOWNLOAD_OPTIONS=""
-ARG LIBERTY_SHA=2e27858abf32abb6b6b61caf3a78fafe15472fb4
+ARG LIBERTY_SHA={replace_with_correct_sha}
```

Comment:
Can the placeholder be replaced with actual SHA value (similar to the other Dockerfiles)?